delegator.py

```diff
@@ -55,15 +56,18 @@ def std_out(self):
 
     @property
     def _pexpect_out(self):
-        result = ''
+        if self.subprocess.encoding:
+            result = ''
+        else:
+            result = bytes()
```

Comment:
This is more stylistic, but I'd prefer we went with `b''` here to represent a byte string. It's more consistent with the coding style of the rest of the KennethReitz projects.